src/global.d.ts

```diff
@@ -13,6 +13,7 @@ interface Window {
   ClipboardItem: any;
   __EXCALIDRAW_SHA__: string | undefined;
   EXCALIDRAW_ASSET_PATH: string | undefined;
+  EXCALIDRAW_EXPORT_SOURCE: string;
```

Comment:
should we rename to `EXCALIDRAW_EXPORT_SOURCE_PATH` for consistency ?

Reply:
sounds good to me!

Reply:
it's not exactly a path :)

Reply:
Should we rename it back to `EXCALIDRAW_EXPORT_SOURCE`

Reply:
let's rename it back, though both are not perfectly correct but since "source" could be any string so "EXCALIDRAW_EXPORT_SOURCE" is better 🙂